__tests__/app.test.js

```diff
@@ -26,3 +27,23 @@ describe("GET /api", () => {
       });
   });
 });
+describe("GET /api/topics", () => {
+  test("200: Responds with an object of all the topics", () => {
```

Comment:
This test description is a little confusing as it is an array of topics that comes back from the endpoint.